backend/api/Controllers/RobotController.cs

```diff
@@ -50,4 +50,25 @@ public async Task<ActionResult<Robot>> PostRobot([FromBody] Robot robot)
             throw;
         }
     }
+
+    [HttpPost]
+    [Route("{robotId}/start/{missionId}")]
+    public async Task<ActionResult<Report>> StartMission([FromRoute] string robotId, [FromRoute] string missionId)
+    {
+        try
+        {
+            var robot = await _robotService.Read(robotId);
+            if (robot == null)
+            {
+                return NotFound();
+            }
+            var newReport = await _robotService.StartMission(robot, missionId);
```

Comment:
Should we also verify that the mission exists somewhere and return a NotFound if it doesn't?

Reply:
Will restructure slightly. Also `_robotService.StartMission(...` will throw an exception if unable to start mission